packages/image/src/image-optimize.ts

```diff
@@ -232,6 +232,18 @@ const DEFAULT_SIZES = "(min-width: 1280px) 50vw, 100vw";
 
 const DEFAULT_QUALITY = 80;
 
+/**
+ * URL.canParse(props.src)
+ */
+export const UrlCanParse = (url: string) => {
```

Comment:
can we please have a better function name? "canParseUrl" would be nice

Reply:
Current one is catchy 😉